src/components/policy/policy_external/test/sql_pt_ext_representation_test.cc

```diff
@@ -264,163 +297,65 @@ ::testing::AssertionResult IsValid(const policy_table::Table& table) {
-  app_policies["default"]["groups"] = Json::Value(Json::arrayValue);
-  app_policies["default"]["groups"][0] = Json::Value("default");
-  app_policies["default"]["priority"] = Json::Value("EMERGENCY");
-  app_policies["default"]["default_hmi"] = Json::Value("FULL");
-  app_policies["default"]["keep_context"] = Json::Value(true);
-  app_policies["default"]["steal_focus"] = Json::Value(true);
-  app_policies["pre_DataConsent"] = Json::Value(Json::objectValue);
-  app_policies["pre_DataConsent"]["memory_kb"] = Json::Value(50);
-  app_policies["pre_DataConsent"]["heart_beat_timeout_ms"] = Json::Value(100);
-  app_policies["pre_DataConsent"]["groups"] = Json::Value(Json::arrayValue);
-  app_policies["pre_DataConsent"]["groups"][0] = Json::Value("default");
-  app_policies["pre_DataConsent"]["priority"] = Json::Value("EMERGENCY");
-  app_policies["pre_DataConsent"]["default_hmi"] = Json::Value("FULL");
-  app_policies["pre_DataConsent"]["keep_context"] = Json::Value(true);
-  app_policies["pre_DataConsent"]["steal_focus"] = Json::Value(true);
-  app_policies["1234"] = Json::Value(Json::objectValue);
-  app_policies["1234"]["memory_kb"] = Json::Value(50);
-  app_policies["1234"]["heart_beat_timeout_ms"] = Json::Value(100);
-  app_policies["1234"]["groups"] = Json::Value(Json::arrayValue);
-  app_policies["1234"]["groups"][0] = Json::Value("default");
-  app_policies["1234"]["priority"] = Json::Value("EMERGENCY");
-  app_policies["1234"]["default_hmi"] = Json::Value("FULL");
-  app_policies["1234"]["keep_context"] = Json::Value(true);
-  app_policies["1234"]["steal_focus"] = Json::Value(true);
-  app_policies["device"] = Json::Value(Json::objectValue);
-  app_policies["device"]["groups"] = Json::Value(Json::arrayValue);
-  app_policies["device"]["groups"][0] = Json::Value("default");
-  app_policies["device"]["priority"] = Json::Value("EMERGENCY");
-  app_policies["device"]["default_hmi"] = Json::Value("FULL");
-  app_policies["device"]["keep_context"] = Json::Value(true);
-  app_policies["device"]["steal_focus"] = Json::Value(true);
-
-  policy_table::Table update(&table);
-  update.SetPolicyTableType(rpc::policy_table_interface_base::PT_UPDATE);
+  std::shared_ptr<policy_table::Table> snapshot = reps_->GenerateSnapshot();
+
+  auto expected_module_meta = GetDefaultSnapshotModuleMeta(update);
+  auto& snapshot_module_meta = snapshot->policy_table.module_meta;
+  EXPECT_EQ(expected_module_meta.toStyledString(),
+            snapshot_module_meta.ToJsonValue().toStyledString());
+}
+
+TEST_F(
+    SQLPTExtRepresentationTest,
+    GenerateSnapshot_SetMandatoryMetaInfo_MandatoryMetaInfoIsPresentInSnapshot) {
+  policy_table::Table update = LoadPreloadedPT(kSdlPreloadedPtJson);
 
-  // Assert
   ASSERT_TRUE(IsValid(update));
   ASSERT_TRUE(reps_->Save(update));
 
-  // Act
+  const std::string ccpu_version = "4.1.3.B_EB355B";
+  const std::string wers_country_code = "WAEGB";
+  const std::string language = "en-us";
+
```

Comment:
@ychernysheva suggest to move out all this const declarations to the top of the file and use them as a global constants from all unit tests here

Reply:
Done in 774db35